packages/terminal-extension/style/base.css

```diff
@@ -0,0 +1,5 @@
+/* Hide the stub toolbar that appears above terminals */
+
+.jp-Toolbar-micro {
```

Comment:
Should this be a more specific selector?

For example `jp-MainAreaWidget > .jp-Toolbar-micro`?

Reply:
Moved to the application extension's CSS.